src/graphics/context.rs

```diff
@@ -853,7 +855,7 @@ impl GraphicsContext {
             FullscreenType::Windowed => {
                 window.set_fullscreen(None);
                 window.set_decorations(!mode.borderless);
-                window.set_inner_size(mode.actual_size()?);
+                let _ = window.request_inner_size(mode.actual_size()?);
```

Comment:
Shouldn't this line be deleted ?

Reply:
The `let _ =` is just there to ignore the result. I'm guessing `request_inner_size` has useful side effects.